src/sshfs_mount/sftp_server.cpp

```diff
@@ -334,6 +336,24 @@ int mp::SftpServer::mapped_gid_for(const int gid)
     return gid;
 }
 
+int mp::SftpServer::reverse_uid_for(const int uid, const int rev_uid_if_not_found)
+{
+    auto compare_uid_map_value = [uid](std::pair<int, int> p) { return uid == p.second; };
+
+    auto found = std::find_if(uid_map.cbegin(), uid_map.cend(), compare_uid_map_value);
```

Comment:
Why not just construct the lambda directly in the `find_if` function like you've done elsewhere?

Reply:
Done.